lib/axe/cucumber/step.rb

```diff
@@ -27,15 +27,15 @@ class Step
       # optionally specify ruleset via list of comma-separated tags
       (?-x:;? according to: (?<tags>.*?))?
 
-      # optionally specify rules (as comma-separated list of rule ids) to check
```

Comment:
These brackets cause Cucumber to choke, Ruby passes them correctly, but Cucumber seems to do some preprocessing, which is where it fails, while our unit tests, which only test the regex itself still pass.